composer.json

```diff
@@ -32,5 +32,7 @@
             "dev-master": "3.0-dev"
         }
     },
+    "minimum-stability": "dev",
```

Comment:
I forgot that the `composer.lock` was commited in the repo. I need to update the `.travis.yml` to execute the tests for both 1.x and 2.x and adapt the matrix accordingly as the supported PHP versions are not the same.